package.json

```
@@ -60,7 +60,9 @@
     "e2e:serve": "yarn build:e2e && node ./server/e2e-test-server.js",
     "e2e:serve:api": "yarn build:e2e:api && node ./server/e2e-test-server.js",
     "playwright:test:mocked": "docker run -v $PWD:/test -w=/test $(node -e \"if(process.env.CI !== 'true'){if(process.platform === 'win32'){console.log('-it -u pwuser')}else{console.log('-it -u $(id -u):$(id -g)')}}else{console.log('-e CI')}\") --rm --ipc=host -p 9323:9323 --add-host host.docker.internal:host-gateway mcr.microsoft.com/playwright:v$(yarn info @playwright/test --name-only --json | sed -n 's/^.*:\\(.*\\)\"$/\\1/p') yarn playwright test",
-    "playwright:test:real": "docker run -v $PWD:/test -w=/test -e \"USE_REAL_API=true\" $(node -e \"if(process.env.CI !== 'true'){if(process.platform === 'win32'){console.log('-it -u pwuser')}else{console.log('-it -u $(id -u):$(id -g)')}}else{console.log('-e CI')}\") --rm --ipc=host -p 9323:9323 --add-host host.docker.internal:host-gateway mcr.microsoft.com/playwright:v$(yarn info @playwright/test --name-only --json | sed -n 's/^.*:\\(.*\\)\"$/\\1/p') yarn playwright test",
+    "playwright:test:mocked:ui": "docker run -it -v $PWD:/test -w=/test $(node -e \"if(process.env.CI !== 'true'){if(process.platform === 'win32'){console.log('-it -u pwuser')}else{console.log('-it -u $(id -u):$(id -g)')}}else{console.log('-e CI')}\")  --rm --ipc=host -p 8080:8080 -p 3000:3000 --add-host host.docker.internal:host-gateway mcr.microsoft.com/playwright:v$(yarn info @playwright/test --name-only --json | sed -n 's/^.*:\\(.*\\)\"$/\\1/p') yarn playwright test --ui --ui-host=0.0.0.0 --ui-port=8080",
```

This command can by tidied up as it doesn't need to check for CI (plus we end up specifying `-it` twice)

```suggestion
    "playwright:test:mocked:ui": "docker run -it -v $PWD:/test -w=/test $(node -e \"if(process.platform === 'win32'){console.log('-u pwuser')}else{console.log('-u $(id -u):$(id -g)')}\")  --rm --ipc=host -p 8080:8080 -p 3000:3000 --add-host host.docker.internal:host-gateway mcr.microsoft.com/playwright:v$(yarn info @playwright/test --name-only --json | sed -n 's/^.*:\\(.*\\)\"$/\\1/p') yarn playwright test --ui --ui-host=0.0.0.0 --ui-port=8080",
```